src/config/configLoader.ts

```diff
@@ -0,0 +1,62 @@
+import { IParserResult } from '@stoplight/types';
+import { existsSync } from 'fs-extra';
+import { isEmpty, merge } from 'lodash';
+import * as path from 'path';
+import { readParsable } from '../fs/reader';
+import { ConfigFormat, IConfig } from '../types/config';
+
+const DEFAULT_CONFIG_FILE = 'spectral.yml';
+
+const loadConfig = async (filePath: any): Promise<IConfig> => {
+  switch (path.extname(filePath)) {
+    case '.json':
+    case '.yaml':
+    case '.yml':
+      const { data, diagnostics }: IParserResult = await readParsable(filePath, 'utf8');
+      if (!isEmpty(diagnostics)) {
+        throw new Error('Parsing exception');
+      }
+      return data;
+    default:
+      throw new Error('Unknown file');
+  }
+};
+
+const extend = async (config: IConfig, filePath: string): Promise<IConfig> => {
```

Comment:
Hey what’s this extends stuff about? The only extends I know about is for rulesets.

Reply:
That's extension that was there since previous PR. I asked later if I should remove it but don't remember clear response. But I can get rid of it easily, just let me know.
One other thing that might be interesting for last call is the default file in cwd. Currently it recognizes only spectral.yml. I can add also json an yaml.

Reply:
Ok, let’s get rid of it. We gotta be careful about accidentally adding features without planning them, or we end up in tricky situations because somebody will use whatever is added and then we’re stuck with it. :)

Reply:
It's removed :)